src/main/java/dataloader/CsvDataLoader.java

```diff
@@ -33,36 +35,66 @@
          * Create a NumericsSample from the given column of the
          * in the TreeMap
          */
-        public static NumericSample buildSample(Map<String, List<String>> dataSet, String colName){
+        public static NumericSample buildNumericSample(Map<String, List<String>> dataSet, String colName){
 
             if(dataSet == null){
 
                 throw new IllegalArgumentException("Null data set given");
             }
 
-            NumericSample sample;
+            NumericSample numericSample;
 
             if(!dataSet.containsKey(colName)){
 
                 if(Configuration.ENABLE_WARNINGS) {
                     Configuration.Logging.printWarning("Column " + colName + " not in dataset");
                 }
 
-                sample =  new NumericSample(colName, 0);
+                numericSample = new NumericSample(colName, 0);
+
             }
             else{
 
                 List<Double> data = ParseUtils.parseAsDouble( dataSet.get(colName) );
-                sample = new NumericSample(colName, data, false);
+                numericSample = new NumericSample(colName, data, false);
             }
+            return  numericSample;
+        }
+
+
+        /**
+         * Create a NumericsSample from the given column of the
+         * in the TreeMap
```

Comment:
pls check the documentation. remove `NumericSample` and add `CategoricalSample`

Reply:
should be fixed in next commit